package-generator/other.sh

```diff
@@ -1 +1,2 @@
 python package-generator.py -u admin -g False -t 00000000000003228460960090160000 -p XXX
```

Comment:
why do not use $1, $2 and maybe $3 for user, password and tenant id and pass these parameters in the CLI?